.github/workflows/release.yml

```
@@ -20,6 +20,6 @@ jobs:
           git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.1.0
+        uses: helm/chart-releaser-action@v1.6.0
```

Should we switch to the actual Git SHA here?
https://github.com/helm/chart-releaser-action/releases/tag/v1.6.0
```suggestion
        uses: helm/chart-releaser-action@a917fd15b20e8b64b94d9158ad54cd6345335584 # 1.6.0
```

---

Yes, good idea! I also did it for the checkout action.